hydroflow/src/scheduled/graph.rs

```diff
@@ -147,21 +203,39 @@ impl Hydroflow {
     pub async fn recv_events_async(&mut self) -> Option<NonZeroUsize> {
         loop {
             let sg_id = self.event_queue_recv.recv().await?;
-            if !self.subgraphs[sg_id].is_scheduled.replace(true) {
-                self.ready_queue.push_back(sg_id);
+            let sg_data = &self.subgraphs[sg_id];
+            if !sg_data.is_scheduled.replace(true) {
+                self.ready_queue[sg_data.stratum].push_back(sg_id);
 
                 // Enqueue any other immediate events.
                 return Some(NonZeroUsize::new(self.try_recv_events() + 1).unwrap());
             }
         }
     }
 
+    pub fn add_subgraph<Name, R, W, F>(
```

Comment:
It would be nice if there was a simple semantic for explicitly vs. non-explicitly stratified subgraphs. This one (where they are always at stratum 0) seems fine to me, but another approach that sounds equally reasonable to me is that non-explicitly stratified operators are always eligible to run? Not sure if we have a preference but seems like something we should decide on.

Reply:
Put in #96 for now